lib/ssz_types/beacon_chain/deposit.ex

```diff
@@ -9,6 +9,11 @@ defmodule SszTypes.Deposit do
     :data
   ]
 
+  @schema [
+    %{proof: %{type: :list, schema: %{type: :bytes, size: 32}, max_size: 33, is_variable: false}},
+    %{data: %{type: :struct, schema: SszTypes.DepositData.schema, schema_struct: SszTypes.DepositData}}
```

Comment:
Isn't it kind of redundant to have both `schema` and `schema_struct`? I think we should drop `schema_struct`.